src/main/java/de/minebench/syncinv/SyncInv.java

```diff
@@ -225,16 +227,16 @@ public void onEnable() {
                                     if (query.getYoungestServer() == null) {
                                         openInvCommand.onCommand(sender, command, label, args);
                                     } else {
-                                        sender.sendMessage(ChatColor.RED + "Current server does not have newest player data! "
-                                                + ChatColor.GRAY + "Connecting to server " + query.getYoungestServer() + " which has the newest data...");
+                                        sender.sendMessage(Component.text("Current server does not have newest player data! ", NamedTextColor.RED).
+                                                append(Component.text("Connecting to server " + query.getYoungestServer() + " which has the newest data...", NamedTextColor.GRAY)));
                                         connectToServer(((Player) sender).getUniqueId(), query.getYoungestServer());
                                     }
                                 });
                                 if (q == null) {
-                                    sender.sendMessage(ChatColor.RED + "Could not query information from other servers! Take a look at the log for more details.");
+                                    sender.sendMessage(Component.text("Could not query information from other servers! Take a look at the log for more details.", NamedTextColor.RED));
                                 }
                             } else {
-                                sender.sendMessage(ChatColor.RED + "Player not found!");
+                                sender.sendMessage(Component.text("Player not found!", NamedTextColor.RED));
```

Comment:
Could probably mole all these messages to the lang config directly without first hardcoding them as components. Not sure why they weren't part of it already...

Reply:
Was surprised to find them hardcoded as well, but didn't want to touch them in case there was a reason for it.